build/tests/service-groups-test.psh

```diff
@@ -0,0 +1,205 @@
+#! /usr/bin/env pdo-shell
+
+## Copyright 2018 Intel Corporation
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+
+set --conditional -s data -v .
+set --conditional -s save -v .
```

Comment:
What is this for/where used?  I would have expected there would be somewhere some use of ${data} and ${save} but couldn't find neither  (obviously showing also my lack of psh knowledge ....)

Reply:
mostly just boiler plate. it could be removed. but it makes invoking other scripts easier.